package.json

```diff
@@ -228,18 +228,16 @@
     ],
     "keybindings": []
   },
-  "scripts": {
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./node_modules/vscode/bin/test"
-  },
+  "scripts": {},
   "devDependencies": {
     "@types/mocha": "^2.2.32",
     "@types/node": "^6.0.40",
-    "electron": "^1.4.13",
-    "electron-rebuild": "^1.5.7",
+    "electron": "^11.0.0",
+    "electron-rebuild": "^3.2.3",
     "eslint": "^5.15.3",
     "mocha": "^6.0.2",
-    "vscode": "^1.1.35"
+    "@vscode/test-electron": "^1.6.2",
+    "@types/vscode": "^1.46.0"
```

Comment:
This is not directly related to the build automation but was needed to get it to successfully run `npm install` again at all.  It could be removed and handled separately.

Reply:
reverted, unrelated